index.js

```diff
@@ -1,35 +1,18 @@
-require('dotenv').config()
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
why do you disable eslint here ?
import/no-extraneous-dependencies on google
if you had an error from linter, it should'nt be here anymore

Reply:
corrected